source/retryable-reads/tests/etc/generate-handshakeError-tests.py

```diff
@@ -0,0 +1,118 @@
+            - insertOne:
+                document: { _id: 1 }'''
+
+COLLECTION_READ_OPERATIONS = [
+    Operation('aggregate', 'aggregate', 'collection', ['pipeline: []']),
+    # Operation('count', 'count', 'collection', ['filter: {}']), # Deprecated.
+    Operation('countDocuments', 'aggregate', 'collection', ['filter: {}']),
+    Operation('estimatedDocumentCount', 'count', 'collection', []),
+    Operation('distinct', 'distinct', 'collection',
+              ['fieldName: x', 'filter: {}']),
+    Operation('find', 'find', 'collection', ['filter: {}']),
+    # Operation('findOne', 'find', 'collection', ['filter: {}']), # Optional.
+    Operation('listIndexes', 'listIndexes', 'collection', []),
+    Operation('listIndexNames', 'listIndexes', 'collection', []),
+    Operation('createChangeStream', 'aggregate',
+              'collection', ['pipeline: []']),
+]
+COLLECTION_WRITE_OPERATIONS = [
+    Operation('insertOne', 'insert', 'collection', ['document: { x: 1 }']),
+    Operation('insertMany', 'insert', 'collection', [INSERT_MANY_ARGUMENTS]),
+    Operation('deleteOne', 'delete', 'collection', ['filter: {}']),
+    Operation('deleteMany', 'delete', 'collection', ['filter: {}']),
+    Operation('replaceOne', 'update', 'collection', [
+              'filter: {}', 'replacement: { x: 1 }']),
+    Operation('updateOne', 'update', 'collection', [
+              'filter: {}', 'update: { $set: { x: 1 } }']),
+    Operation('updateMany', 'update', 'collection', [
+              'filter: {}', 'update: { $set: { x: 1 } }']),
+    Operation('findOneAndDelete', 'findAndModify',
+              'collection', ['filter: {}']),
+    Operation('findOneAndReplace', 'findAndModify', 'collection',
+              ['filter: {}', 'replacement: { x: 1 }']),
+    Operation('findOneAndUpdate', 'findAndModify', 'collection',
+              ['filter: {}', 'update: { $set: { x: 1 } }']),
+    Operation('bulkWrite', 'insert', 'collection', [BULK_WRITE_ARGUMENTS]),
+    Operation('createIndex', 'createIndexes', 'collection',
+              ['keys: { x: 1 }', 'name: "x_1"']),
+    Operation('dropIndex', 'dropIndexes', 'collection', ['name: "x_1"']),
+    Operation('dropIndexes', 'dropIndexes', 'collection', []),
+]
+COLLECTION_OPERATIONS = COLLECTION_READ_OPERATIONS + COLLECTION_WRITE_OPERATIONS
+
+# Session and GridFS operations are generally tested in other files, so they're not included in the list of all
+# operations. Individual generation functions can choose to include them if needed.
+OPERATIONS = CLIENT_OPERATIONS + DB_OPERATIONS + COLLECTION_OPERATIONS
+
+RETRYABLE_READ_OPERATIONS = [op for op in OPERATIONS if op.operation_name in
+                             ['find', 'findOne', 'aggregate', 'distinct', 'count', 'estimatedDocumentCount', 'countDocuments', 'createChangeStream', 'listDatabases',
+                              'listDatabaseNames', 'listCollections', 'listCollectionNames', 'listIndexes']
+                             ]
```

Comment:
No; `RETRYABLE_READ_OPERATIONS` is a filter of `OPERATIONS`. There are more `OPEARTIONS` than there are `RETRYABLE_READ_OPERATIONS`.

Reply:
Can you name one such operation?

Reply:
I was confusing the current script with the original template where `OPERATIONS` included both read and write operations. This will once again be the case with the use of a single generator script.